src/CameraZoomPlugin.cc

```diff
@@ -154,6 +164,32 @@ class CameraZoomPlugin::Impl
   public: rendering::CameraPtr camera;
 };
 
+/// \brief Convert from focal length to FOV for a rectilinear lens
+/// \ref https://en.wikipedia.org/wiki/Focal_length
+/// @param sensorWidth Diagonal sensor width [meter]
+/// @param focalLength The focal length [meter]
+/// @return The field of view [rad]
+[[nodiscard]] static double FocalLengthToFov(
+  const double& sensorWidth,
+  const double& focalLength);
+
+/// \brief Convert from FOV to focal length for a rectilinear lens
+/// \ref https://en.wikipedia.org/wiki/Focal_length
+/// @param sensorWidth Diagonal sensor width [meter]
+/// @param fov The field of view [rad]
+/// @return The focal length [meter]
+[[nodiscard]] static double FovToFocalLength(
```

Comment:
Do we need the `[[nodiscard]]`. Seems like a policy change, in which case it should be applied consistently and probably as part of a separate PR enforcing this and other C++17 annotations.

Reply:
Nope. I see the build is set to only require C++14; will remove.